src/SBarcodeDecoder.h

```diff
@@ -4,8 +4,15 @@
 #include <QObject>
 #include <QVideoFrame>
 
+#include <QOpenGLContext>
+#include <QOpenGLFunctions>
+
 #include "SBarcodeFormat.h"
 
+// Default camera resolution width/height
+#define DEFAULT_RES_W 1080
+#define DEFAULT_RES_H 1920
```

Comment:
Do not hardcode camera resolution!
Use the information from device only

Reply:
Those are default values and dynamically changed according to the device resolution in the SBarcodeScanner::initCam() method